docs/examples/CustomMultiValueContainer.tsx

```diff
@@ -5,7 +5,7 @@ import { ColourOption, colourOptions } from '../data';
 
 const MultiValueContainer = (props: MultiValueGenericProps<ColourOption>) => {
   return (
-    <Tooltip content={'Customise your multi-value container!'}>
+    <Tooltip content={'Customize your multi-value container!'}>
```

Comment:
"Customise" and its variants are intentional since Jed is from Australia.

Reply:
Reverting those

Reply:
And doing the changes backwards. The intent was to keep either `"customiz**"`or `"customis**`. Changing all to `"customis**"`